aws/ecs_cluster.go

```diff
@@ -0,0 +1,49 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// getAllEcsClusters - Returns a string of ECS Cluster ARNs, which uniquely identifies the cluster.
+// NOTE: AWS api doesn't provide the necessary information to
+//       implement `excludeAfter` filter at the cluster
+//       level, so we will implement it at the service level.
```

Comment:
Wait, so does that mean we delete all ECS clusters, no matter how old they are?

Reply:
There are other resources that don't expose a "created date." For those, we use tags to only delete resources older than a certain date. E.g., see the EIP code: https://github.com/gruntwork-io/cloud-nuke/blob/master/aws/eip.go

Reply:
I was actually going to go that route originally, except in my original search I couldn't find any way to set tags in ECS. But I guess I didn't try hard enough then because now I found that they are called something else in ECS (`Attribute`), which explains why I didn't find it the first time: I was looking for the wrong thing :(

... except unfortunately, it looks like it won't even let you set attributes on a cluster:

```
houston exec sandbox -- aws --region us-east-1 ecs put-attributes --cluster "$CLUSTER_ARN" --attributes name="cloud-nuke-test",value="1234",targetId="$CLUSTER_ARN"

An error occurred (InvalidParameterException) when calling the PutAttributes operation: Unexpected targetType 'cluster'.
```

I can't find anything else in the ECS API suite that lets me set custom values on a resource. The only thing we have control of is the `Name`, which isn't something we should overload.

Should I just remove the cluster nuking for now?

Reply:
Yea, probably... We should be conservative in a tool like this and never delete more than expected.

Reply:
Make sense. Will take it out.

Reply:
Added a caveat section to the README to signal that we have considered this and made the conscious choice not to pursue this at the moment.